src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -258,15 +258,15 @@ public void deleteUploadSchemaByIdAndRevision(String schemaId, int revision) {
     }
 
     @Override
-    public UploadSchema getUploadSchemaById(String schemaId) {
+    public UploadSchema getUploadSchema(String schemaId) {
         session.checkSignedIn();
         checkArgument(isNotBlank(schemaId), Bridge.CANNOT_BE_BLANK, "schemaId");
         return get(config.getUploadSchemaByIdApi(schemaId), UploadSchema.class);
     }
 
     @Override
-    public ResourceList<UploadSchema> getUploadSchemaForStudy() {
+    public ResourceList<UploadSchema> getUploadSchemasAllRevisions() {
```

Comment:
getAllUploadSchemasAllRevisions()
getAllUploadSchemaRevisions()
getAllRevisionsOfAllUploadSchemas()
?
every researcher call is for a study, so it isn't conveying to me that it's getting the schemas in two dimensions (all revisions of all schemas)